bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/action/LGWebOSActions.java

```diff
@@ -200,12 +202,18 @@ public void sendButton(
                 case BACK:
                     getConnectedSocket().ifPresent(control -> control.executeMouse(s -> s.button(ButtonType.BACK)));
                     break;
+                case EXIT:
+                    getConnectedSocket().ifPresent(control -> control.executeMouse(s -> s.button(ButtonType.EXIT)));
+                    break;
                 case DELETE:
                     getConnectedSocket().ifPresent(control -> control.sendDelete());
                     break;
                 case ENTER:
                     getConnectedSocket().ifPresent(control -> control.sendEnter());
                     break;
+                case ENTER_NAV:
+                    getConnectedSocket().ifPresent(control -> control.executeMouse(s -> s.button(ButtonType.ENTER)));
+                    break;
```

Comment:
Not your code, but in line 218 below `"HOME"` could be replaced by `ButtonType.HOME`.

Reply:
I removed the restriction on values.